File: Python/lista1 - estrutura de decisao/exercicio_9.py
year_of_birth = int(input('Insira o seu ano do seu nascimento: '))
difference = 2022 - year_of_birth

if difference >= 18:
    print('Possui idade para votar e ter CNH.')
elif difference >= 16 and difference <= 17:
    print('Possui idade para votar. Não pode ter CNH.')
else:
    print('Não possui idade para votar e nem ter CNH.')

File: C/lista1 - estrutura de decisao/exercicio_9.c
#include <conio.h>
#include <stdio.h>

int main() {
	int yearOfBirth, difference;
	
	printf("Informe o ano de nascimento: ");
	scanf("%d", &yearOfBirth);
	
	difference = 2022 - yearOfBirth;
	
	if (difference >= 18) {
		printf("Idade para votar e tirar CNH.");
	} else if (difference >= 16 && difference <= 17) {
		printf("Idade para votar, mas ainda nao pode tirar CNH.");
	} else {
		printf("Nao tem idade para votar, nem para tirar CNH.");
	}
	
	getch();
	return 0;
}